### ISSUE TITLE
Imbalance data-set handling

### ISSUE THREAD

**Issue by ShibaPrasad** [fizyr/keras-retinanet#828]
Hi,
I am using retina-net model for my detection problem. But I have a small question on my training dataset
We have around 5500 image data sets of two classes. But they are imbalanced. 

How to handle the very imbalance dataset on retinanet?

I read about Focal Loss on the retina-net paper and not sure whether it addresses the above data imbalance issue or not. (Would like to clarify more elaborately)

**Comment by hgaiser** [comment]
Focal loss is better equipped to deal with imbalance, but the best solution would be no imbalance. There isn't really anything to do in keras-retinanet to have less effect from the imbalanced datasets. Simply try it out and see if the results are satisfactory.

Closing this as it is not an issue with the codebase. Please use the slack channel (see README) for usage information.
